release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1527,6 +1539,12 @@ In this update, `systemd` service only sets the default RPS mask for virtual int
 [id="ocp-4-11-storage-bug-fixes"]
 ==== Storage
 
+* Previously, the Alibaba Container Storage Interface (CS) driver that shipped with {product-title} returned errors when a user created a `PersistentVolumeClaim` (PVC) smaller than 20 GiB. This was due to Alibaba Cloud only supporting volumes larger than 20 GiB. With this update, Alibaba CSI driver automatically increases all volume sizes to at least 20 GiB and smaller PVCs are now dynamically provisioned. This can result in increased costs. Administrators can use quota on PVC count for each namespace in restricted environments to limit costs. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2057495[*BZ#2057495*])
+
+* In earlier versions, the Local Storage Operator (LSO) added an OwnerReference to the Persistent Volumes (PVs) it created, such that deletion of the node would also issue a delete request for the PV. This could result in the PV remaining in the "Terminating" state while still attached to a pod. LSO no longer creates that OwnerReference, which means cluster administrators must delete any unused PVs after a node is removed from the cluster. For more information, see "Persistent storage using local volumes". (link:https://bugzilla.redhat.com/show_bug.cgi?id=2061447#[*BZ#2061447*])
```

Comment:
owner reference

persistent volumes (PVs)

    `Terminating` state

Add an xref to https://docs.openshift.com/container-platform/4.10/storage/persistent_storage/persistent-storage-local.html